data-raw/db_info_table.csv

```diff
@@ -26,3 +26,4 @@ rxpand,2020-10-20,1,https://raw.githubusercontent.com/jgregoriods/rxpand/master/
 agrichange,2021-05-21,1,https://zenodo.org/record/4541470/files/AgriChange_14Cdatabase.xlsx
 caribbean,2021-05-21,1,https://raw.githubusercontent.com/philriris/caribbean-14C/main/data/caribbean_14C.csv
 aida,today,1,https://raw.githubusercontent.com/apalmisano82/AIDA/main/dates.csv
+p3k14c,2021-07-13,1,https://github.com/people3k/p3k14c/raw/0.9.1/data/p3k14c_data.rda
```

Comment:
It's a bit risky to store data in an .rda file, because it's not human-readable and they change sometimes between R versions. Do they have a text file or an excel table (:roll_eyes:)?